webrender/src/device/gfx/device.rs

```diff
@@ -179,15 +179,15 @@ pub struct Device<B: hal::Backend> {
     pub depth_format: hal::format::Format,
     pub queue_group_family: QueueFamilyId,
     pub queue_group_queues: Vec<B::CommandQueue>,
-    command_pools: SmallVec<[CommandPool<B>; FRAME_COUNT_MAILBOX]>,
+    command_pools: ArrayVec<[CommandPool<B>; FRAME_COUNT_MAILBOX]>,
```

Comment:
Why don't we have a `Frame` structure (array of them) with all these fields as opposed to multiple arrays?

Reply:
Now we have: 0978ae85776536bcfcacc2ac13f318d1d39deefc